internal/db/applications.go

```diff
@@ -90,12 +101,11 @@ func GetApplicationsByOrgID(ctx context.Context, orgID uuid.UUID) ([]types.Appli
 
 func GetApplicationsWithLicenseOwnerID(ctx context.Context, id uuid.UUID) ([]types.Application, error) {
 	db := internalctx.GetDb(ctx)
-	// TODO: Only include versions from at least one license
 	if rows, err := db.Query(ctx, `
-			SELECT DISTINCT `+applicationWithVersionsOutputExpr+`
+			SELECT DISTINCT `+applicationWithLicensedVersionsOutputExpr+`
 			FROM ApplicationLicense al
 				LEFT JOIN Application a ON al.application_id = a.id
-			WHERE al.owner_useraccount_id = @id
+			WHERE al.owner_useraccount_id = @id AND (al.expires_at IS NULL OR al.expires_at > now())
```

Comment:
I would 
1. do this in a separate PR
2. probably do it in application code and show it as "expired" or something instead, hiding like this could cause unforeseen problems

Reply:
this only applies if the license feature is active and the user is not a `vendor` (see application handler)

Reply:
(and the license is already checked, only the expiration date was ignored)